spring-cloud-stream-binders/spring-cloud-stream-binder-rabbit/src/main/java/org/springframework/cloud/stream/binder/rabbit/RabbitMessageChannelBinder.java

```diff
@@ -232,6 +232,8 @@ public MessageProperties toMessageProperties(AMQP.BasicProperties source, Envelo
 
 	private volatile int defaultTxSize = DEFAULT_TX_SIZE;
 
+	protected volatile boolean defaultDurableSubscription = true;
+
```

Comment:
I am fine with the status quo of this PR - but perhaps we can introduce some notion of 'optional' properties that binders **may** support (instead of making it a rabbit-specific property) - it might be as simple as an interface with constants.